external/pom.xml

```diff
@@ -23,6 +23,11 @@
             <version>1.9.7</version>
         </dependency>
         <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>wikipediaAPI-multilingual</artifactId>
+            <version>1.1.12</version>
+        </dependency>
+        <dependency>
```

Comment:
mainly this is why I need it. I could extract the MultilingualTokenizer, I guess...